src/lib.rs

```diff
@@ -216,24 +224,22 @@ impl<'a> PullRequest<'a> {
             self.pull_number.unwrap()
         );
         let response = fetch_github_data(&fetch_params, RequestMethod::GET).await;
-        match response {
-            Ok(data) => {
-                return Ok::<Value, FreshEyesError>(data);
-            }
+        return match response {
+            Ok(data) => Ok::<Value, FreshEyesError>(data),
             Err(e) => {
                 if let FreshEyesError::StatusCodeError(error_response) = e {
-                    if error_response.status == StatusCode::NOT_FOUND.as_u16() {
-                        return Err(FreshEyesError::StatusCodeError(ErrorResponse {
-                            message: format!("pull request not found!").to_string(),
+                    return if error_response.status == StatusCode::NOT_FOUND.as_u16() {
+                        Err(FreshEyesError::StatusCodeError(ErrorResponse {
+                            message: "pull request not found!".to_string().to_string(),
```

Comment:
why are we calling `to_string()` twice?

Reply:
quite unneccessary, must be an over sight. will remove.